src/main/java/lotto/dto/LottoNumbersDto.java

```diff
@@ -0,0 +1,22 @@
+package lotto.dto;
+
+import lotto.LottoNumber;
+import lotto.LottoNumbers;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoNumbersDto {
+    private final Set<LottoNumber> lottoNumbers;
```

Comment:
```suggestion
    private final Set<LottoNumberDto> lottoNumbers;
```
해당 도메인을 그대로 리턴하면 큰 의미가 없을거 같아요

Reply:
혹은 그냥 LottoNumberDto를 없애고
```java
private final Set<Integer> lottoNumbers;
```
를 사용하는 방식도 있겠네요

Reply:
> 혹은 그냥 LottoNumberDto를 없애고
> 
> ```java
> private final Set<Integer> lottoNumbers;
> ```
> 
> 를 사용하는 방식도 있겠네요

아! 머리를 탁 쳤네요 감사합니다!